libvast/src/schema.cpp

```diff
@@ -183,4 +185,13 @@ bool convert(const schema& s, json& j) {
   return true;
 }
 
+caf::expected<schema> load_schema_file(const path& sf) {
+  if (sf.empty())
+    return make_error(ec::filesystem_error, "");
```

Comment:
```suggestion
    return make_error(ec::filesystem_error, "empty path");
```

Reply:
Actually, isn't `load_contents` taking care of the case when loading from file doesn't work?

Reply:
This is just code moved from `reader_command.hpp` to reuse if from `load_schema_dirs()`
`load_contents()` doesn't seem to handle that case correctly, I think it would just return an empty string.